main.ts

```diff
@@ -1,11 +1,11 @@
 import fetch from 'node-fetch';
-import * as fs from 'fs';
+import {URLSearchParams} from 'url';
 
 const pathToUltsky = 'D:/Games/UltSky'
 
 async function makeLoadOrder() {
-    const formData = new FormData();
-    const files = document.querySelector('input[type="file"][multiple]');
+    const formData = new URLSearchParams();
+    // const files = document.querySelector('input[type="file"][multiple]');

```

Comment:
I was running this via node because there wasn't a frontend to run. `document` isn't available in node.